src/lib/gameLogic/GameLobby.js

```diff
@@ -221,6 +224,24 @@ class GameLobby {
     return Math.floor((this.drawTimer / this.timerLength) * 100);
   }
 
+  updateMessage(message) {
+    this.messages.push(message);
+    console.log('message:', message);
+    // console.log("MESSAGE FROM GAMELOBBY : ", this.messages);
+    let chatMessage = '';
+    if (message.evaluate) {
+      chatMessage = `${message.userName} guessed the word.`;
+    } else {
+      chatMessage = `${message.userName}: ${message.guess}`;
+    }
+    const newChatMessage = new EventOptions({
+      message: {message: chatMessage},
+      event: SocketEvents.USER_MESSAGE,
```

Comment:
think this should be changed to Lobby message which is set up on the front end already to print messages to chat!